pkg/ipamclaimswebhook/podmutator.go

```diff
@@ -233,54 +302,117 @@ func ensureIPAMClaimRefAtNetworkSelectionElements(ctx context.Context,
 	return hasChangedNetworkSelectionElements, nil
 }
 
-func findNewPrimaryNetworkIPAMClaimName(ctx context.Context,
-	cli client.Client, pod *corev1.Pod, vmName string) (string, error) {
+func findPrimaryUDNLogicalNetworkAttachment(
+	ctx context.Context,
+	vmi *virtv1.VirtualMachineInstance,
+	pluginConfig *config.RelevantConfig,
```

Comment:
We no longer need the pluginConfig

Reply:
What do you mean we don't need it ? We return if the plugin doesn't support persistent ips.

Reply:
> What do you mean we don't need it ? We return if the plugin doesn't support persistent ips.

I see the function just returning the VMI, where is the persisent IPs check ? I am talking about `runningVMForPod` fuction.

Reply:
ahh right, I mixed stuff as always,  so with the udn-client update we no longer need to check persisentips right ? so we don't need pluginConfig, or do we ?